### PR TITLE
Fixed bug: Error in Create project from backup for Standard 3D Annotation 

### LINKED ISSUE #4105: Error in Create project from backup for Standard 3D Annotation
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)
- [x] Tested create project from backup for a 2D Annotation project succesfull

I tried to backup and restore a project for a Standard 3D annotation mode and the import of the backup failed.
I was able to export the backup without any issues.

### Expected Behaviour

The project should be proberly recreated from the backup

### Current Behaviour

After selecting the zipped backup and load it, an error is raised:
Could not restore the project
Error: Request failed with status code 500. "Exception: Invalid manifest was uploaded\n".

### Steps to Reproduce (for bugs)

1. Clone the repo and build the docker images from source from actual commit on develop branch (see commit hash bellow)
2. Create a demo project (just on label at this moment)
3. Create a new task for Standard 3D annotation with following demo sample structure (Task creation was succesfull and Pointcloud and context image are displayed correctly): 
data/
  im1/
    im.pcd
    imp.png
4. Add a single demo annotation.
5. Export Backup of the project
6. Delete the original demo project
7. Tried to import the backup to create the project again.
8. The import failed with following error message in the UI:
  Could not restore the project
  Error: Request failed with status code 500. "Exception: Invalid manifest was uploaded\n".

### Context

Try to test the backup process befor starting larger projects.

### Your Environment

- Git hash commit (`git log -1`):
  commit f915385f2c7c10b7004aee55d524f37514658c43 (HEAD -> develop, origin/develop, origin/HEAD)
  Author: AmitN1212 <69145846+AmitN1212@users.noreply.github.com>
  Date:   Fri Dec 31 07:08:00 2021 +0200

      Update documentation (Development Environment section) (#4082)
- Docker version `docker version` (e.g. Docker 17.0.05):
  Docker version 20.10.11, build dea9396
- Are you using Docker Swarm or Kubernetes?
  No
- Operating System and version (e.g. Linux, Windows, MacOS):
  Distributor ID: Ubuntu
  Description:    Ubuntu 20.04.3 LTS
  Release:        20.04
  Codename:       focal
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container
^[[36mcvat          |^[[0m 2022-01-04 01:46:01,492 DEBG 'rqworker_default_0' stderr output:
^[[36mcvat          |^[[0m INFO - 2022-01-04 01:46:01,492 - worker - default: cvat.apps.engine.backup._import_project('/tmp/cvat_p7f_jegy', 2, 1) (User@/api/v1/projects/cfd57a53-9bab-4197-bf03-f27219c99a61/import)
^[[36mcvat          |^[[0m
^[[36mcvat          |^[[0m 2022-01-04 01:46:01,492 DEBG 'rqworker_default_0' stderr output:
^[[36mcvat          |^[[0m DEBUG - 2022-01-04 01:46:01,492 - worker - Sent heartbeat to prevent worker timeout. Next one should arrive within 480 seconds.
^[[36mcvat          |^[[0m
^[[36mcvat          |^[[0m 2022-01-04 01:46:01,501 DEBG 'rqworker_default_0' stderr output:
^[[36mcvat          |^[[0m DEBUG - 2022-01-04 01:46:01,501 - worker - Sent heartbeat to prevent worker timeout. Next one should arrive within 90 seconds.
^[[36mcvat          |^[[0m
^[[36mcvat          |^[[0m 2022-01-04 01:46:01,549 DEBG 'rqworker_default_0' stderr output:
^[[36mcvat          |^[[0m [2022-01-04 01:46:01,549] INFO cvat.server: create task #32
^[[36mcvat          |^[[0m
^[[36mcvat          |^[[0m 2022-01-04 01:46:01,549 DEBG 'rqworker_default_0' stderr output:
^[[36mcvat          |^[[0m INFO - 2022-01-04 01:46:01,549 - task - create task #32
^[[36mcvat          |^[[0m
^[[36mcvat          |^[[0m 2022-01-04 01:46:01,551 DEBG 'rqworker_default_0' stderr output:
^[[36mcvat          |^[[0m [2022-01-04 01:46:01,551] WARNING cvat.server: Skip 'index.json' file (its mime type doesn't correspond to supported MIME file type)
^[[36mcvat          |^[[0m
^[[36mcvat          |^[[0m 2022-01-04 01:46:01,551 DEBG 'rqworker_default_0' stderr output:
^[[36mcvat          |^[[0m WARNING - 2022-01-04 01:46:01,551 - task - Skip 'index.json' file (its mime type doesn't correspond to supported MIME file type)
^[[36mcvat          |^[[0m
^[[36mcvat          |^[[0m 2022-01-04 01:46:01,553 DEBG 'rqworker_default_0' stderr output:
^[[36mcvat          |^[[0m DEBUG - 2022-01-04 01:46:01,553 - worker - Handling failed execution of job user@/api/v1/projects/cfd57a53-9bab-4197-bf03-f27219c99a61/import
^[[36mcvat          |^[[0m
^[[36mcvat          |^[[0m 2022-01-04 01:46:01,555 DEBG 'rqworker_default_0' stderr output:
^[[36mcvat          |^[[0m ERROR - 2022-01-04 01:46:01,555 - worker - Traceback (most recent call last):
^[[36mcvat          |^[[0m   File "/opt/venv/lib/python3.8/site-packages/rq/worker.py", line 936, in perform_job
^[[36mcvat          |^[[0m     rv = job.perform()
^[[36mcvat          |^[[0m   File "/opt/venv/lib/python3.8/site-packages/rq/job.py", line 684, in perform
^[[36mcvat          |^[[0m     self._result = self._execute()
^[[36mcvat          |^[[0m   File "/opt/venv/lib/python3.8/site-packages/rq/job.py", line 690, in _execute
^[[36mcvat          |^[[0m     return self.func(*self.args, **self.kwargs)
^[[36mcvat          |^[[0m   File "/usr/lib/python3.8/contextlib.py", line 75, in inner
^[[36mcvat          |^[[0m     return func(*args, **kwds)
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/backup.py", line 652, in _import_project
^[[36mcvat          |^[[0m     db_project = project_importer.import_project()
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/backup.py", line 644, in import_project
^[[36mcvat          |^[[0m     self._import_tasks()
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/backup.py", line 634, in _import_tasks
^[[36mcvat          |^[[0m     TaskImporter(
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/backup.py", line 527, in import_task
^[[36mcvat          |^[[0m     self._import_task()
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/backup.py", line 510, in _import_task
^[[36mcvat          |^[[0m     _create_thread(self._db_task.pk, data.copy(), True)
^[[36mcvat          |^[[0m   File "/usr/lib/python3.8/contextlib.py", line 75, in inner
^[[36mcvat          |^[[0m     return func(*args, **kwds)
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/task.py", line 262, in _create_thread
^[[36mcvat          |^[[0m     manifest_file = _validate_manifest(manifest_files, manifest_root)
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/task.py", line 203, in _validate_manifest
^[[36mcvat          |^[[0m     raise Exception('Invalid manifest was uploaded')
^[[36mcvat          |^[[0m Exception: Invalid manifest was uploaded
^[[36mcvat          |^[[0m Traceback (most recent call last):
^[[36mcvat          |^[[0m   File "/opt/venv/lib/python3.8/site-packages/rq/worker.py", line 936, in perform_job
^[[36mcvat          |^[[0m     rv = job.perform()
^[[36mcvat          |^[[0m   File "/opt/venv/lib/python3.8/site-packages/rq/job.py", line 684, in perform
^[[36mcvat          |^[[0m     self._result = self._execute()
^[[36mcvat          |^[[0m   File "/opt/venv/lib/python3.8/site-packages/rq/job.py", line 690, in _execute
^[[36mcvat          |^[[0m     return self.func(*self.args, **self.kwargs)
^[[36mcvat          |^[[0m   File "/usr/lib/python3.8/contextlib.py", line 75, in inner
^[[36mcvat          |^[[0m     return func(*args, **kwds)
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/backup.py", line 652, in _import_project
^[[36mcvat          |^[[0m     db_project = project_importer.import_project()
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/backup.py", line 644, in import_project
^[[36mcvat          |^[[0m     self._import_tasks()
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/backup.py", line 634, in _import_tasks
^[[36mcvat          |^[[0m     TaskImporter(
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/backup.py", line 527, in import_task
^[[36mcvat          |^[[0m     self._import_task()
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/backup.py", line 510, in _import_task
^[[36mcvat          |^[[0m     _create_thread(self._db_task.pk, data.copy(), True)
^[[36mcvat          |^[[0m   File "/usr/lib/python3.8/contextlib.py", line 75, in inner
^[[36mcvat          |^[[0m     return func(*args, **kwds)
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/task.py", line 262, in _create_thread
^[[36mcvat          |^[[0m     manifest_file = _validate_manifest(manifest_files, manifest_root)
^[[36mcvat          |^[[0m   File "/home/django/cvat/apps/engine/task.py", line 203, in _validate_manifest
^[[36mcvat          |^[[0m     raise Exception('Invalid manifest was uploaded')
^[[36mcvat          |^[[0m Exception: Invalid manifest was uploaded
</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

### PR DESCRIPTION
**Pull request by Marishka17**
### Motivation and context

Resolve #4105

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
~~- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly~~
- [x] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
~~- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2022 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat/apps/engine/task.py [hunk 1]
```diff
@@ -326,13 +326,15 @@ def _create_thread(db_task, data, isBackupRestore=False, isDatasetImport=False):
...
+            if isBackupRestore and media_type != 'video' and db_data.storage_method == models.StorageMethodChoice.CACHE and \
+                    db_data.sorting_method in {models.SortingMethod.RANDOM, models.SortingMethod.PREDEFINED}:
                 # we should sort media_files according to the manifest content sequence
                 manifest = ImageManifestManager(db_data.get_manifest_path())
                 manifest.set_index()
                 sorted_media_files = []
-                for idx in range(len(media_files)):
-                    properties = manifest[manifest_index(idx)]
+
+                # FIXME it's bad solution because a manifest file can contain more files then were uploaded
```
Comment (nmanovic) [inline cvat/apps/engine/task.py:336]: @Marishka17 , what are we going to do with the comment?

File: utils/dataset_manifest/core.py [hunk 2]
```diff
@@ -706,12 +706,15 @@ def _validate_first_item(_dict):
             raise ValueError('Incorrect name field')
         if not isinstance(_dict['extension'], str):
             raise ValueError('Incorrect extension field')
-        # width and height are required for 2d data
-        # FIXME for 3d when manual preparation of the manifest will be implemented
-        if not isinstance(_dict['width'], int):
-            raise ValueError('Incorrect width field')
-        if not isinstance(_dict['height'], int):
-            raise ValueError('Incorrect height field')
+        # FIXME
```
Comment (nmanovic) [inline utils/dataset_manifest/core.py:709]: @Marishka17 , the same question. What are we going to do with the comment? It looks like we disable two checks which can be important for 2D case.

Comment (Marishka17) [reply utils/dataset_manifest/core.py:709]: @nmanovic,  yes, but these checks are only relevant for the first element of the manifest file, and the reliability will drop a bit...

**Review by nmanovic — APPROVED** [review]
LGTM

**Pull request merged**